src/DependenciesPackager/Program.cs

```diff
@@ -184,7 +189,7 @@ static int Main(string[] args)
                 packagesVersion,
                 useCli,
                 quiet,
-                runtimes,
+                runtime,
```

Comment:
Confirmed with coreclr team. Crossgen has to be done on the same target platform, which means send in multiple runtimes is essentially useless. The only exception is that for Windows, it is supported to crossgen 32bits assembly on 64bits assembly.

My motivation of changing this parameter from multiple value to single value is to 1) eliminate incorrect assumption and 2) simplify work flow.